src/main/resources/application.properties

```diff
@@ -2,4 +2,9 @@ server.port=9090
 
 server.servlet.context-path=/staff
 
+logging.file.max-size:10MB
+logging.file.max-history=2
+logging.file=E:\myworkspace\staff_management\log.txt
```

Comment:
it would be more appropriate to move this file property to environment-specific properties.